.circleci/config.yml

```diff
@@ -7,20 +7,24 @@ orbs:
 commands:
   setup_tools:
     description: "Install Homebrew and Ruby dependencies"
+    parameters:
+      skip_brew_install:
+        type: boolean
+        default: false
     steps:
       - restore_cache:
           name: Restore Homebrew + Ruby Dependencies
           keys:
             - &cache_key brew-dependencies-{{ checksum ".circleci/.brewfile" }}-{{ checksum "Gemfile.lock" }}
-
-      - run:
-          name: Install Homebrew dependencies, if neeeded
-          command: brew update && xargs brew install --verbose < .circleci/.brewfile
-
+      - unless:
+          condition: <<parameters.skip_brew_install>>
+          steps:
+            - run:
+                name: Install Homebrew dependencies, if neeeded
+                command: brew update && xargs brew install --verbose < .circleci/.brewfile
```

Comment:
This is nice – ideally we could get rid of the need for `brew` entirely before long?

Reply:
Yeah I'm not entirely sure about this because I have assumed that the `test` job needs imagemagick to at least run the tests / `rspec` which probably include some tests about running `drawText` or testing some screenshot-related actions, which is why it would need those?